### ISSUE TITLE
cholesterol gallblader polyp

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11067]
**Preferred term label:**
cholesterol gallblader polyp

**Synonyms**

**Definition (free text, please give PubMed ID)**
A non-neoplastic gallbladder polyp that is hyperplasticity and consists of a lipid deposit.

**Parent term (use hpo.jax.org/app)**
non-neoplastic gallbladder polyp

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0009-6791-365X

**Comment by pnrobinson** [comment]
@nklsbckmnn  thanks!
Adding
Cholesterol gallblader polyp
HP:6001359
A non-neoplastic gallbladder polyp that is hyperplastic and consists of a lipid deposit. Cholesterol polyps are generally less than 10 mm. often multiple cholesterol polyps are present. They are often associated with vesicular cholesterolosis and are thought to have no malignant potential.
PMID: 22655278
